packages/netlify-cms-core/index.d.ts

```diff
@@ -462,16 +462,16 @@ declare module 'netlify-cms-core' {
     fieldsMetaData: Map<string, any>;
   }
 
-  export interface CmsWidgetParam {
+  export interface CmsWidgetParam<T = any> {
```

Comment:
Thanks @iChenLei! Does `T = any` means this work break for people who were using the old version of type?

Reply:
```txt
![2021-12-03 19 56 55](https://user-images.githubusercontent.com/14012511/144598707-5c61ee8e-630a-4273-b3d0-e7a726cde865.png)
![2021-12-03 19 57 30](https://user-images.githubusercontent.com/14012511/144598769-466294b6-51c8-40dc-b199-486547adf293.png)
![2021-12-03 19 58 00](https://user-images.githubusercontent.com/14012511/144598833-a0f5f415-4403-4bc4-b6d8-f71e373d6f01.png)
```

> assumes that the value of value in the control and preview should always be the same, but I think that's the case?  --usernamenumber 

Sorry, sir. I didn't notice the comment like above. So maybe this is a invalid pull request. Because I don't know should the `CmsWidgetControlProps` amd `CmsWidgetPreviewProps`'s `T` be same ?

> Btw, `T = any` didn't break anything, because the default type in `typescript` is `any`.

Reply:
```ts
export interface CmsWidgetParam<T = any, S = any> {
    name: string;
    controlComponent: CmsWidgetControlProps<T> ;
    previewComponent?: CmsWidgetPreviewProps<S>;
    globalStyles?: any;
}
```
Is this the right solution ?

Reply:
Thanks @iChenLei, `CmsWidgetControlProps` and `CmsWidgetPreviewProps` should receive the same type so the original version is good

Reply:
Mr.erezrokah, thanks for guide ! I will fix CI problem this night.

Reply:
No worries, I already fixed the CI, just waiting for the tests to finish